cmd/bbolt/command_check.go

```diff
@@ -4,25 +4,72 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 
 	bolt "go.etcd.io/bbolt"
 	"go.etcd.io/bbolt/internal/guts_cli"
 )
 
 func newCheckCommand() *cobra.Command {
 	checkCmd := &cobra.Command{
-		Use:   "check <bbolt-file>",
-		Short: "verify integrity of bbolt database data",
+		Use:   "check <subcommand>",
+		Short: "check related commands",
+	}
+
+	checkCmd.AddCommand(newCheckDBCommand())
+	checkCmd.AddCommand(newCheckPageCommand())
+
+	return checkCmd
+}
+
+func newCheckDBCommand() *cobra.Command {
+	checkDBCmd := &cobra.Command{
+		Use:   "check-db <bbolt-file>",
```

Comment:
I think `bbolt check check-db ...`, and `check check-page ..`, sound redundant. Would it be good to remove the `check-` prefix?

Reply:
👍🏽